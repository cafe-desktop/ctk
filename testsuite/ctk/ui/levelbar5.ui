<!-- invalid element in levelbar markup -->
<interface>
  <object class="CtkLevelBar">
    <offsets>
      <warbl/>
    </offsets>
  </object>
</interface>
