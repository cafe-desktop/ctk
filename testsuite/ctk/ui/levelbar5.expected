ERROR: ctk-builder-error-quark 1
.:5:15 Unsupported tag for CtkLevelBar: <warbl>
